### ISSUE TITLE
[@foal/jwt] Invalid import path

### ISSUE THREAD

**Issue by yeager-j** [FoalTS/foal#328]
**Describe the bug**

The files `jwt-optional.hook.d.ts` and `jwt-required.hook.d.ts` seem to have bad import paths.

```
node_modules/@foal/jwt/lib/jwt-optional.hook.d.ts(3,91): error TS2307: Cannot find module '../../../../../../../../Users/loicpoullain/projects/FoalTS/foal/packages/jwt/node_modules/@foal/core/lib/core/hooks'.
node_modules/@foal/jwt/lib/jwt-required.hook.d.ts(3,91): error TS2307: Cannot find module '../../../../../../../../Users/loicpoullain/projects/FoalTS/foal/packages/jwt/node_modules/@foal/core/lib/core/hooks'.
```

**To Reproduce**
1. Create fresh project
2. `$ yarn add @foal/jwt`
3. Run program

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] *Mac OS Mojave*
 - Version [e.g. 22] *0.7.3*

**Additional context**

**Comment by LoicPoullain** [comment]
Hi @yeager-j,

Thanks for reporting this!

I got the same issue yesterday. Looks like the library didn't compile properly when releasing the package.

Weirdly, deleting the node_modules directory and running again the "npm install" makes the "npm run develop" command to work sometimes.

Anyway, I fixed the import error in this PR: #325 . I'll release the new version today.

**Comment by LoicPoullain** [comment]
Fixed in v0.7.6
